Sources/VLCMediaViewEditCell.swift

```diff
@@ -0,0 +1,131 @@
+        view.layer.borderColor = UIColor.lightGray.cgColor
+        view.layer.borderWidth = 1
+        return VLCCheckView(isEnabled: false, view: view)
+    }()
+
+    let thumbnail: UIImageView = {
+        let thumbnail = UIImageView()
+        thumbnail.translatesAutoresizingMaskIntoConstraints = false
+        thumbnail.contentMode = .scaleAspectFit
+        thumbnail.clipsToBounds = true
+        thumbnail.layer.cornerRadius = 3
+        return thumbnail
+    }()
+
+    let title: UILabel = {
+        let title = UILabel()
+        title.textColor = PresentationTheme.current.colors.cellTextColor
+        title.font = UIFont.systemFont(ofSize: 17)
+        title.translatesAutoresizingMaskIntoConstraints = false
+        return title
+    }()
+
+    let subInfo: UILabel = {
+        let subInfo = UILabel()
+        subInfo.textColor = PresentationTheme.current.colors.cellTextColor
+        subInfo.font = UIFont.systemFont(ofSize: 13)
+        subInfo.translatesAutoresizingMaskIntoConstraints = false
+        return subInfo
+    }()
+
+    let size: UILabel = {
+        let size = UILabel()
+        size.textColor = PresentationTheme.current.colors.cellTextColor
+        size.font = UIFont.systemFont(ofSize: 11)
+        size.translatesAutoresizingMaskIntoConstraints = false
+        return size
+    }()
+
+    let mainStackView: UIStackView = {
+        let mainStackView = UIStackView()
+        mainStackView.spacing = 20.0
+        mainStackView.axis = .horizontal
+        mainStackView.alignment = .center
+        mainStackView.translatesAutoresizingMaskIntoConstraints = false
+        return mainStackView
+    }()
+
+    let mediaInfoStackView: UIStackView = {
+        let mediaInfoStackView = UIStackView()
+        mediaInfoStackView.spacing = 5.0
+        mediaInfoStackView.axis = .vertical
+        mediaInfoStackView.alignment = .leading
+        mediaInfoStackView.translatesAutoresizingMaskIntoConstraints = false
+        return mediaInfoStackView
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupViews()
+    }
+
+    private func setupViews() {
+
+        mediaInfoStackView.addArrangedSubview(title)
+        mediaInfoStackView.addArrangedSubview(subInfo)
+        mediaInfoStackView.addArrangedSubview(size)
+
+        mainStackView.addArrangedSubview(checkView.view)
+        mainStackView.addArrangedSubview(thumbnail)
+        mainStackView.addArrangedSubview(mediaInfoStackView)
+
+        addSubview(mainStackView)
+
+        var guide: LayoutAnchorContainer = self
+
+        if #available(iOS 11.0, *) {
+            guide = safeAreaLayoutGuide
+        }
+        NSLayoutConstraint.activate([
+            checkView.view.heightAnchor.constraint(equalToConstant: 20),
+            checkView.view.widthAnchor.constraint(equalTo: checkView.view.heightAnchor),
```

Comment:
The checkview should resize itself according to the image

Reply:
Not according to figma, is there a specific reason why?

Reply:
this has less to do with figma but more with how the VLCCheckMark view should behave.

Reply:
Why should it resize? The only reason I possibly see is accessibility.

Reply:
Easier to maintain in the future if we change the image for example or Apple does something crazy.